src/Atomic.hpp

```diff
@@ -39,12 +39,21 @@ inline uint64_t AtomicAdd(uint64_t *ptr, uint64_t value)
 #endif // TUNDRA_WIN32_MINGW
 }
 
+inline void* AtomicCompareExchange(void **ptr, void* newPtr, void* comparePtr)
+{
+#if defined(TUNDRA_WIN32_MINGW)
+    #error not implemented
```

Comment:
Does this mean that mingw is unsupported from now on, or is this just temporary?

Reply:
Yes, we don't support this for now. Do we need to? If we have any use for this, it's probably easy to fix. But without a way to even test it, I'd rather put in an error, then some code which might work, and probably doesn't.